src/Microsoft.NetCore.Analyzers/UnitTests/Security/ReviewCodeForSqlInjectionVulnerabilitiesTests.cs

```diff
@@ -0,0 +1,2852 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.NetCore.Analyzers.Security.UnitTests
+{
+    using System;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Microsoft.NetCore.Analyzers.Security;
+    using Test.Utilities;
+    using Xunit;
+
+    public class ReviewCodeForSqlInjectionVulnerabilitiesTests : DiagnosticAnalyzerTestBase
+    {
+        protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
+        {
+            return new ReviewCodeForSqlInjectionVulnerabilities();
+        }
+
+        protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
+        {
+            return new ReviewCodeForSqlInjectionVulnerabilities();
+        }
+
+        protected DiagnosticResult GetCSharpResultAt(int sinkLine, int sinkColumn, int sourceLine, int sourceColumn, string sink, string sinkContainingMethod, string source, string sourceContainingMethod)
+        {
+            this.PrintActualDiagnosticsOnFailure = true;
+            return GetCSharpResultAt(
+                new[] {
+                    Tuple.Create(sinkLine, sinkColumn),
+                    Tuple.Create(sourceLine, sourceColumn)
+                },
+                ReviewCodeForSqlInjectionVulnerabilities.Rule,
+                sink,
+                sinkContainingMethod,
+                source,
+                sourceContainingMethod);
+        }
+
+        protected void VerifyCSharpWithDependencies(string source, params DiagnosticResult[] expected)
+        {
+            this.VerifyCSharp(source, ReferenceFlags.AddTestReferenceAssembly, expected);
+        }
+
+        [Fact]
+        [Trait(Traits.DataflowAnalysis, Traits.Dataflow.TaintedDataAnalysis)]
```

Comment:
>[Trait(Traits.DataflowAnalysis, Traits.Dataflow.TaintedDataAnalysis)] [](start = 8, length = 69)

Nit: Consider moving the trait to the class, so future tests are automatically included for the trait.